sci-physics/hepmc/hepmc-3.2.5.ebuild

```
@@ -0,0 +1,51 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# python3_11 fails
+PYTHON_COMPAT=( python3_{8..10} )
+inherit cmake python-single-r1
+
+MYP=HepMC3-${PV}
+
+DESCRIPTION="Event Record for Monte Carlo Generators"
+HOMEPAGE="https://hepmc.web.cern.ch/hepmc/"
+SRC_URI="https://hepmc.web.cern.ch/hepmc/releases/${MYP}.tar.gz"
+S="${WORKDIR}/${MYP}"
+
+LICENSE="GPL-3+"
+SLOT="3"
+KEYWORDS="~amd64"
+IUSE="doc test examples python root"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="python? ( ${PYTHON_DEPS} )"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	root? ( sci-physics/root:= )
+	doc? (
+		app-doc/doxygen
```

```suggestion
		app-doc/doxygen[dot]
```
*      sh: line 1: dot: command not found